tests/rbd/rbd_purge_cluster.py

```diff
@@ -0,0 +1,53 @@
+from cli.cephadm.cephadm import CephAdm
```

Comment:
We already have a module tests/cephadm/test_remove_cluster.py for purging the ceph cluster. 
@sunilangadi2  Don't you think we can reuse the same? Instead of creating a separate module?

Reply:
yes i removed this module as not needed, purge will be successful using the playbook command hence used `test_cephadm_ansible.py` this existing module to purge ceph.